### ISSUE TITLE
Unify casing

### ISSUE THREAD

**Issue by Twixes** [PostHog/posthog-js#60]
### Proposal
Currently original parts of the code use snake_case, while newer additions use pascalCase. It's JS convention though to use pascalCase in most cases and it might be a good idea to refactor at least exported things (that users see) with that convention in mind.

### Upsides
The current mix doesn't look very professional and might be leaving a poor impression on developers wanting to use PostHog.

### Downsides
This maneuver would leave quite a lot of these snake_case parts and nuking them is not really an option at this point, so they'd have to be left functioning, just deprecated – that wouldn't be good for code readability.

**Comment by mariusandra** [comment]
Yeah, this is definitely something to do one day! We should have a good [test suite](https://github.com/PostHog/posthog-js/issues/50) before embarking on this journey though.
